### ISSUE TITLE
Show example of .complete("unknown") 

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/browser-payment-api#584]
As part of https://github.com/w3c/payment-request/pull/583, we've somewhat struggled to come up with a real use case for completing a payment response with "unknown". 

Although "unknown" feels like a good idea, can anyone think of a *real* use case where:
 
* somehow you get into the "unknown" situation? 
* "fail" doesn't suffice? 

If we can't come up with something useful, we should probably mark it "at risk". 

Also noting that Chrome (on desktop) currently treats `.complete("unknown")` the same as `.complete("success")`, in that it just closes the payment sheet and doesn't inform the user about anything. 

I'm not sure yet what we will do in Firefox. It will depend on the outcome of this discussion, I guess.   

In the spec, "unknown" plays no role in any algorithm.

**Comment by marcoscaceres** [comment]
Talked to @zkoch, he mentioned 3D Secure can lead to "unknown".

**Comment by zkoch** [comment]
@marcoscaceres , are you OK to close this issue?

**Comment by marcoscaceres** [comment]
Was going to add something to the .complete() example.  I'll update the bug title to not confuse people.

**Comment by dsajmr-commercial** [comment]
Dsa here... Looking to provide an update, trust me it's necessary at this point. What's the proper forum, how to go about it and what are the parameters... I'm sure our values var-y by millions of bytes... Let's open up a private chat where discussion can take place in plain English. I'm not a coder in case you haven't guessed.

**Comment by ianbjacobs** [comment]
@marcoscaceres, can we close this issue now?

Ian
